### PR TITLE
Remove unused dependencies

### PR DESCRIPTION
**Pull request by thomaseizinger**
Thanks cargo-udeps!

### CODE REVIEW

File: cnd/Cargo.toml [hunk 1]
```diff
@@ -34,11 +31,10 @@ http-api-problem = "0.13"
 hyper = "0.12"
 lazy_static = "1"
 libp2p = { version = "0.13" }
-libp2p-core = { version = "0.13" }
```
Comment (D4nte) [inline cnd/Cargo.toml]: How come it's here? don't we have tomlfmt running in the CI?

Comment (thomaseizinger) [reply cnd/Cargo.toml]: Should actually, not know why it slipped through :man_shrugging:

**Comment by tcharding** [comment]
`cargo-udeps` cool, good thing I didn't try to hard to write one.

File: cnd/Cargo.toml (hunk 1 above)
Comment (D4nte) [reply cnd/Cargo.toml]: I think you fixed it, for tomlfmt you have to explicitly pass each cargo toml file.

**Pull request merged**